sigle/src/pages-lib/_error.tsx

```diff
@@ -1,38 +1,136 @@
-import React from 'react';
+import React /*{ useEffect }*/ from 'react';
+import { useEffect } from 'react';
 import { NextPageContext } from 'next';
 import NextErrorComponent from 'next/error';
-import styled from 'styled-components';
-import tw from 'twin.macro';
+import { styled } from '../stitches.config';
 import Link from 'next/link';
 import * as Sentry from '@sentry/nextjs';
-import { Button, Container } from '../components';
-import { sigleConfig } from '../config';
+import { Button, Text } from '../ui';
+import { Container } from '../ui';
+import Image from 'next/image';
+import { useAuth } from '../modules/auth/AuthContext';
 
-const NotFoundContainer = styled(Container)`
-  ${tw`mt-8 mb-8 flex flex-col items-center text-center md:flex-row md:text-left`};
-`;
+const NotFoundContainer = styled(Container, {
+  '@xs': {
+    flexDirection: 'column',
+    textAlign: 'center',
+  },
 
-const NotFoundTextContainer = styled(Container)`
-  ${tw`flex flex-col items-center md:items-start md:pl-8`};
-`;
+  '@lg': {
+    flexDirection: 'row-reverse',
+    textAlign: 'left',
+  },
 
-const NotFoundTitle = styled.div`
-  ${tw`mb-2 text-5xl`};
-`;
+  px: 0,
+  width: '100%',
+  display: 'flex',
+  alignItems: 'center',
+  mx: 'auto',
+  mt: '$8',
+  mb: '$3',
+});
 
-const NotFoundSubTitle = styled.div`
-  ${tw`mb-8 text-3xl`};
-`;
+const NotFoundTextContainer = styled(Container, {
```

Comment:
Is there a reason this one is extending the `Container` instead of just being a `div`? I know it was like this before but probably just my mistake at this time :p